src/test/regress/sql/ddl_create.sql

```diff
@@ -0,0 +1,19 @@
+CREATE SCHEMA ddl_test_schema;
+SET search_path TO ddl_test_schema;
+
+CREATE TABLE test_tbl (a INT, b text, c BIGINT);
+SELECT create_distributed_table('test_tbl','a');
+
+CREATE INDEX CONCURRENTLY a_index ON test_tbl (a);
+
+BEGIN;
```

Comment:
Some ideas, not that I'm not suggesting to add 10 tests, but as much as use the same indexes and expand these index parameters.

1) create a partitioned table, have multiple indexes. One on the parent, one on the child etc. Also, use `ONLY` keyword as well?

2) Maybe use expressions on the indexes? `CREATE INDEX index_on_child ON sensors_2020_01_01(lower(measure_data::text));`

3) Maybe use a non-btree index: `CREATE INDEX hash_index ON sensors USING HASH((measure_data->'IsFailed'));` or GIN etc.

4) Maybe change the statistics of the index? `ALTER INDEX index_on_parent ALTER COLUMN 1 SET STATISTICS 1000;` 

5) Maybe cluster on the pkey ?`CLUSTER colocated_dist_table USING colocated_dist_table_pkey;`

6) Maybe create a replica identity over an index? `ALTER TABLE index_backed_rep_identity REPLICA IDENTITY USING INDEX uqx;`


7) Maybe create index with `WHERE` clause?  (e.g., partial index)
8) Maybe add `storage_parameter `  and `COLLATE` to the index?

9) Maybe use `ASC , DESC, NULLS FIRST` on the columns with multiple columns
10) Maybe use `IF NOT EXISTS` syntax